pkg/controller/kcc/kcct.go

```
@@ -338,6 +346,31 @@ func (k *KatalystCustomConfigTargetController) syncKCCTs(gvr metav1.GroupVersion
 		return utilerrors.NewAggregate(errors)
```

There might be a bug that should return an error if the length of `errors` is greater than 0, and return nil if the length of `targetResources` equals 0.